CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ endif()
 
 # For non-supported platforms, turn USE_DISTRIBUTED off by default.
 # It is not tested and likely won't work without additional changes.
-if(NOT LINUX)
+if(APPLE)
```

Comment:
this means we plan to turn on distributed package by default for Windows? Is there anything Windows-specific instructions that we need to add to README.md?

Reply:
Here I changed it to APPLE is to avoid setting USE_DISTRIBUTED back to OFF when user setting USE_DISTRIBUTED to ON in environment variable. So if user dose not set USE_DISTRIBUTED to ON in environment variable, by default distributed package will not be enabled on Windows.